pkgs/development/libraries/brainflow/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "brainflow";
+  version = "5.10.2";
+
+  src = fetchFromGitHub {
+    owner = "brainflow-dev";
+    repo = "brainflow";
+    rev = "${version}";
+    sha256 = "sha256-8SZDma05imXcCW/NbRjQTkxL9gKSDCQ64yG6E+GhevA=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  meta = with lib; {
+    description = "BrainFlow is a library intended to obtain, parse and analyze EEG, EMG, ECG and other kinds of data from biosensors.";
+    homepage = "https://brainflow.org/";
+    license = licenses.mit;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ ziguana ];
```

Comment:
We usually order the attributes alphabetically.

Reply:
Moved `platforms` down, I think that's what you meant.